cyences_app_for_splunk/default/transforms.conf

```diff
@@ -34,103 +34,51 @@ external_type = kvstore
 
 [cs_sophos_inventory]
 external_type = kvstore
 collection = cs_sophos_inventory_collection
-fields_list = _key, time, ip, hostname, mac_address, sophos_uuid, service_status, sophos_customer_id, sophos_health, sophos_lockdown_status, sophos_lockdown_update_status, sophos_login_via, sophos_os, sophos_os_details, sophos_product_installed, sophos_services_status, sophos_threats_status, sophos_type, sophos_user, tamperProtectionEnabled
+fields_list = _key, time, indextime, product_name, product_uuid, ip, hostname, mac_address, sophos_uuid, service_status, sophos_customer_id, sophos_health, sophos_lockdown_status, sophos_lockdown_update_status, sophos_login_via, sophos_os, sophos_os_details, sophos_product_installed, sophos_services_status, sophos_threats_status, sophos_type, sophos_user, tamperProtectionEnabled
 
 [cs_windows_defender_inventory]
 external_type = kvstore
 collection = cs_windows_defender_inventory_collection
-fields_list = _key, time, ip, hostname, mac_address, windows_defender_host, AVSignature_version, BM_state, Engine_version, IOAV_state, Last_full_scan_end_time, Last_full_scan_start_time, Last_quick_scan_end_time, Last_quick_scan_start_time, OA_state, Platform_version, RTP_state
+fields_list = _key, time, indextime, product_name, product_uuid, ip, hostname, mac_address, windows_defender_host, AVSignature_version, BM_state, Engine_version, IOAV_state, Last_full_scan_end_time, Last_full_scan_start_time, Last_quick_scan_end_time, Last_quick_scan_start_time, OA_state, Platform_version, RTP_state
 
 [cs_crowdstrike_inventory]
 external_type = kvstore
 collection = cs_crowdstrike_inventory_collection
-fields_list = _key, time, ip, hostname, mac_address, crowdstrike_userid, crowdstrike_collected_by, crowdstrike_customer_id
+fields_list = _key, time, indextime, product_name, product_uuid, ip, hostname, mac_address, crowdstrike_userid, crowdstrike_collected_by, crowdstrike_customer_id
```

Comment:
In these lookups, 2-3 new fields are added and rest of the fields are as is. so on upgrade,
- existing data will be as is and new fields will be empty for existing data.
- when scheduled search executes, it override the existing entries if _key is matched. (we kept _key same for these lookups i.e. product_uuid)
- the entries which are not overridden will be cleaned up when "device inventory cleanup" search run.  (because it filters data based on indextime hence existing entries with empty indextime will be cleaned up.)